### PR TITLE
/DAMP/VREL - SPMD fix

### PR DESCRIPTION
**Pull request by bertrand-altair**
Fix of SPMD and PARITH/ON issue for /DAMP/VREL + fix of uninitialized variable

### CODE REVIEW

File: engine/source/assembly/damping_vref.F [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      USE GROUPDEF_MOD
+C-----------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include      "implicit_f.inc"
+#include      "comlock.inc"
+C-----------------------------------------------
+C   C o m m o n   B l o c k s
+C-----------------------------------------------
+#include      "com01_c.inc"
```
Comment (sebastienVilleneuve) [inline engine/source/assembly/damping_vref.F:44]: you can not use commons anymore

File: engine/source/assembly/dampvref_sum6.F [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+C-----------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include      "implicit_f.inc"
+#include      "comlock.inc"
+C-----------------------------------------------
+C   C o m m o n   B l o c k s
+C-----------------------------------------------
+#include      "com01_c.inc"
+#include      "com04_c.inc"
```
Comment (sebastienVilleneuve) [inline engine/source/assembly/dampvref_sum6.F:45]: commons use no more allowed

File: starter/source/restart/ddsplit/c_dampvrel.F [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+C   M o d u l e s
+C-----------------------------------------------
+      USE GROUPDEF_MOD     
+C---------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include      "implicit_f.inc"
+#include      "param_c.inc"
+#include      "com01_c.inc"
+#include      "com04_c.inc"
```
Comment (sebastienVilleneuve) [inline starter/source/restart/ddsplit/c_dampvrel.F:44]: not possible to add commons include anymore

File: starter/source/restart/ddsplit/w_dampvrel.F [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+C   M o d u l e s
+C-----------------------------------------------
+      USE GROUPDEF_MOD     
+C---------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include      "implicit_f.inc"
+#include      "param_c.inc"
+#include      "com01_c.inc"
+#include      "com04_c.inc"
```
Comment (sebastienVilleneuve) [inline starter/source/restart/ddsplit/w_dampvrel.F:22]: commons not allowed in new routines

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]

File: engine/source/assembly/damping_vref.F (hunk 1 above)
Comment (bertrand-altair) [reply engine/source/assembly/damping_vref.F]: commons removed

File: engine/source/assembly/dampvref_sum6.F (hunk 2 above)
Comment (bertrand-altair) [reply engine/source/assembly/dampvref_sum6.F]: commons removed

File: starter/source/restart/ddsplit/c_dampvrel.F (hunk 3 above)
Comment (bertrand-altair) [reply starter/source/restart/ddsplit/c_dampvrel.F]: commons removed

File: starter/source/restart/ddsplit/w_dampvrel.F (hunk 4 above)
Comment (bertrand-altair) [reply starter/source/restart/ddsplit/w_dampvrel.F]: commons removed

**Review by sebastienVilleneuve — APPROVED** [review]

**Pull request merged**